incubator/hnc/internal/kubectl/describe.go

```diff
@@ -57,7 +57,7 @@ var describeCmd = &cobra.Command{
 			if _, ok := childrenAndStatus[cn]; ok {
 				childrenAndStatus[cn] = "subnamespace"
 			} else {
-				childrenAndStatus[cn] = "missing subnamespace"
+				childrenAndStatus[cn] = "missing subnamespace; may be an excluded namespace"
```

Comment:
sg. Updated and it will look like:
```
$ kubectl hns describe parent
Hierarchy configuration for namespace parent
  No parent
  Children:
  - d (Conflict subnamespace)
  - e (subnamespace)
  - f
  - kube-system (Forbidden subnamespace)
  No conditions
```